criu/crtools.c

```diff
@@ -69,8 +69,10 @@ int main(int argc, char *argv[], char *envp[])
 	BUG_ON(get_service_fd(SERVICE_FD_MIN+1) <
 	       get_service_fd(SERVICE_FD_MAX-1));
 
-	if (fault_injection_init())
+	if (fault_injection_init()) {
+		pr_err("CRIU_FAULT out of bounds when initializing fault injection.\n");
```

Comment:
Unfortunately, criu/fault_injection.c does not include any log.h files, preventing us from putting this more specific error message inside `fault_injection_init` instead. Should we keep it like this, update fault_injection.c, or revert?

Reply:
I think including log.h in fault_injection.c is a good idea